apps/etl-func/README.md

```diff
@@ -1 +1,45 @@
 # etl-func
+
+## How to run locally
+
+### Create a local.settings.json
+
+You can use the `local.settings.json.example` file:
+
+```bash
+cp local.settings.json.example ./.local.settings.json
+```
+
+### Add your test environment variables
+
+t.b.d.
+
+```json
+#TODO: Add environment variables here
```

Comment:
test env variables should be provided in a `local.settings.json.example` file

Reply:
[cd5f5bf](https://github.com/pagopa/io-messages/pull/50/commits/cd5f5bfbc728d55d2ccb934024f2f1da32adde79)